File: FE/lib/model/model_qltvn/lop_hoc.dart
class LopHoc {
  int? id;
  String? name;
  String? khoa;
  String? tenChuNhiem;
  String? sdtChuNhiem;
  String? emailChuNhiem;
  int? status;

  LopHoc({
    this.id,
    this.name,
    this.khoa,
    this.tenChuNhiem,
    this.sdtChuNhiem,
    this.emailChuNhiem,
    this.status,
  });

  LopHoc.fromJson(Map<String, dynamic> json)
      : id = json['id'],
        name = json['name'],
        khoa = json['khoa'],
        tenChuNhiem = json['tenChuNhiem'],
        sdtChuNhiem = json['sdtChuNhiem'],
        emailChuNhiem = json['emailChuNhiem'],
        status = json['status'];

  Map<String, dynamic> toJson() => {'name': name, 'khoa': khoa, 'tenChuNhiem': tenChuNhiem, 'sdtChuNhiem': sdtChuNhiem, 'emailChuNhiem': emailChuNhiem, 'status': status};
}

File: BE/src/main/java/com/haivn/common_api/LopHoc.java
package com.haivn.common_api;

import lombok.Getter;
import lombok.Setter;
import org.hibernate.annotations.DynamicUpdate;
import org.hibernate.annotations.Where;

import javax.persistence.*;

@Entity
@Table(name = "lop_hoc")
@Getter
@Setter
@DynamicUpdate
@Where(clause = "deleted=false")
public class LopHoc extends BaseEntity{
    @Column(name = "name")
    private String name;
    @Column(name = "khoa")
    private String khoa;
    @Column(name = "ten_chu_nhiem")
    private String tenChuNhiem;
    @Column(name = "sdt_chu_nhiem")
    private String sdtChuNhiem;
    @Column(name = "email_chu_nhiem")
    private String emailChuNhiem;
    @Column(name = "status")
    private Short status;

}
